.github/workflows/pytorch-version-tests.yml

```diff
@@ -34,6 +34,10 @@ jobs:
             python-version: 3.9
           - pytorch-version: 1.8.1
             python-version: 3.9
+          - pytorch-version: 1.3.1
+            python-version: 3.10
+          - pytorch-version: 1.4.0
+            python-version: 3.10
```

Comment:
Have you checked if 1.5.1, 1.6.0 etc have 3.10 python binaries ?

Reply:
Let me check
When I test with github action, it tests with `3.1` not `3.10`. May I ask why it does not recognize `10` ??
https://github.com/puhuk/ignite/actions/runs/3249191243/jobs/5331308987

Reply:
it should be a str "3.10"

Reply:
I check from this page and I exclude all the pytorch version with python `3.10` except `1.11.0` and `1.12.1`